cli/src/template_builder.rs

```diff
@@ -2334,6 +2346,58 @@ mod tests {
             @"[38;5;1mtext[39m");
     }
 
+    #[test]
+    fn test_raw_escape_sequence_function() {
+        // Note: similar to test_label_function above
+        let mut env = TestTemplateEnv::new();
+        env.add_keyword("empty", || L::wrap_boolean(Literal(true)));
+        env.add_color("error", crossterm::style::Color::DarkRed);
+        env.add_color("warning", crossterm::style::Color::DarkYellow);
+
+        // Literal
+        insta::assert_snapshot!(
+            env.render_ok(r#"raw_escape_sequence(label("error", "text"))"#), @"text");
+
+        // Evaluated property
+        insta::assert_snapshot!(
+            env.render_ok(r#"raw_escape_sequence(label("error".first_line(), "text"))"#),
+            @"text");
+
+        // Template
+        insta::assert_snapshot!(
+            env.render_ok(r#"raw_escape_sequence(label(if(empty, "error", "warning"), "text"))"#),
+            @"text");
```

Comment:
nit: I don't think these tests are needed (because they test the behavior of `label()`, not `raw_escape_sequence()`.)

The "Literal" one should be good enough. `label(..)` is not a literal but a template argument passed to `raw_escape_sequence(..)`.

Reply:
Hmm, "strips labels" is part of raw_escape_sequence's contract, hence this test making sure the output is stripped of them, why do you say they test behaviour of label?

Reply:
I mean the first example `raw_escape_sequence(label("error", "text"))` covers the others. It doesn't matter whether the label name expression is literal or not.

Reply:
Ah, that makes sense, thanks!